src/main/java/database/ArangoHandler.java

```diff
@@ -2,103 +2,117 @@
-
-            /* If database does not exist */
-            if (!driver.getDatabases().contains(databaseName)) {
-                /* Create database */
-                driver.createDatabase(databaseName);
-                /* Create collection */
-                driver.db(databaseName).createCollection(collectionName);
-                System.out.println("Database created");
-            }
-
-            /* Select database */
-            database = driver.db(databaseName);
-            /* Select collection */
-            collection = database.collection(collectionName);
-        } catch (IOException e){
-            e.printStackTrace();
+        String databaseName = configProps.getProperty("db.name");
+        String collectionName = configProps.getProperty("collection.users.name");
+
+        /* Select driver */
+        arangoDriver = getDBConnection().getArangoDriver();
+
+        /* If database does not exist */
+        if (!arangoDriver.getDatabases().contains(databaseName)) {
+            /* Create database */
+            arangoDriver.createDatabase(databaseName);
+            /* Create collection */
+            arangoDriver.db(databaseName).createCollection(collectionName);
+            System.out.println("Database created");
         }
 
+        /* Select database */
+        database = arangoDriver.db(databaseName);
+        /* Select collection */
+        collection = database.collection(collectionName);
     }
 
     /**
      * Disconnects from ArangoDB
      */
     public void disconnect() {
         /* Tear down connection */
-        driver.shutdown();
-    }
-
-    /**
-     * Retrieves user profile by key
-     */
-    public UserProfile getUserProfile(String key) {
-        /* Retrieve user profile */
-        return collection.getDocument(key, UserProfile.class);
+        arangoDriver.shutdown();
     }
 
     /**
      * Creates user profile
      */
-    public String createUserProfile(UserProfile userProfile) throws ArangoDBException {
-        /* Check validity before insertion */
-        if (!isValidUserProfile(userProfile)) {
-            throw new RuntimeException("Invalid user profile");
-        }
+    public String createUser(Object user) {
+        UserProfile userProfile= (UserProfile) user;
 
-        /* Insert user profile */
         DocumentCreateEntity insertedUserProfile = collection.insertDocument(userProfile);
         return insertedUserProfile.getKey();
     }
 
+
     /**
-     * Validates user profile
+     * Retrieves user profile by key
      */
-    public boolean isValidUserProfile(UserProfile userProfile) {
-        // TODO: Check if valid email
-        // TODO: Check if email does not already exist
-        return true;
+    public Object getUser(String id) {
+        /* Retrieve user profile */
+        return collection.getDocument(id, UserProfile.class);
     }
 
     public static void main(String[] args) {
```

Comment:
We don't need a main method on dev, if you need it later make TODO to delete it at the end.

Reply:
ok